apps/studio/pages/project/[ref]/settings/auth.tsx

```diff
@@ -1,53 +1,89 @@
-        </ScaffoldHeader>
-      </ScaffoldContainer>
-      <ScaffoldContainer className="flex flex-col gap-10" bottomPadding>
-        {!isPermissionsLoaded ? (
-          <GenericSkeletonLoader />
-        ) : !canReadAuthSettings ? (
-          <NoPermission isFullPage resourceText="access your project's authentication settings" />
-        ) : (
-          <>
-            <BasicAuthSettingsForm />
-            <SmtpForm />
-            <AdvancedAuthSettingsForm />
-            <ThirdPartyAuthForm />
-          </>
-        )}
-      </ScaffoldContainer>
-    </>
+    <ProductEmptyState title="Authentication settings have moved">
+      <div className="text-sm">
+        <p className="text-foreground-light mb-4">
+          All settings are now under configuration within the Authentication page. All other product
+          settings will follow in the coming weeks.
+        </p>
+        <Link
+          href={`/project/${ref}/auth/providers`}
+          className="block py-2 hover:text-foreground border-b flex items-center justify-between"
+        >
+          General user signup
+          <ChevronRight strokeWidth={1.5} size={16} />
+        </Link>
+        <Link
+          href={`/project/${ref}/auth/providers`}
+          className="block py-2 hover:text-foreground border-b flex items-center justify-between"
+        >
+          Password settings in email provider
+          <ChevronRight strokeWidth={1.5} size={16} />
+        </Link>
+        <Link
+          href={`/project/${ref}/auth/sessions`}
+          className="block py-2 hover:text-foreground border-b flex items-center justify-between"
+        >
+          User sessions and refresh tokens
+          <ChevronRight strokeWidth={1.5} size={16} />
+        </Link>
+        <Link
+          href={`/project/${ref}/auth/protection`}
+          className="block py-2 hover:text-foreground border-b flex items-center justify-between"
+        >
+          Bot and abuse protection
+          <ChevronRight strokeWidth={1.5} size={16} />
+        </Link>
+        <Link
+          href={`/project/${ref}/auth/smtp`}
+          className="block py-2 hover:text-foreground border-b flex items-center justify-between"
+        >
+          SMTP settings
+          <ChevronRight strokeWidth={1.5} size={16} />
+        </Link>
+        <Link
+          href={`/project/${ref}/settings/api`}
+          className="block py-2 hover:text-foreground border-b flex items-center justify-between"
+        >
+          Access token expiry
+          <ChevronRight strokeWidth={1.5} size={16} />
+        </Link>
+        <Link
+          href={`/project/${ref}/auth/mfa`}
+          className="block py-2 hover:text-foreground border-b flex items-center justify-between"
+        >
+          Multi-factor authentication
+          <ChevronRight strokeWidth={1.5} size={16} />
+        </Link>
+        <Link
+          href={`/project/${ref}/settings/database`}
```

Comment:
Max request duration links to database settings - but this has been shifted to auth/advanced

![image](https://github.com/user-attachments/assets/c5043c50-29d0-41af-ab2d-7ca512076547)

Reply:
Also Max Direct Database Connections is missing here - which should be the one routed to settings/database